protected/views/base/_footer.php

```diff
@@ -31,12 +31,14 @@
 		<div class="panel">
 			<ul class="inline-list">
 				<li><strong>Need help?</strong></li>
-				<?php if (Yii::app()->params['helpdesk_email']) { ?>
-					<li><?php echo Yii::app()->params['helpdesk_email'] ?></li>
-				<?php } ?>
-				<?php if (Yii::app()->params['helpdesk_phone']) { ?>
+        <?php
+        $help_email = SettingInstallation::model()->find('`key`="helpdesk_email"')['value'];
+        $help_email .= $help_email !== null ?null:Yii::app()->params['helpdesk_email']; ?>
+        <li><?php echo  $help_email?></li>
+				<?php
+        $help_phone = SettingInstallation::model()->find('`key`="helpdesk_phone"')[value];
+        $help_phone .= $help_email !== null ?null: Yii::app()->params['helpdesk_phone'];?>
```

Comment:
This should be $help_phone. Also fix spacing here and consider inverting the ternary/converting to a coalesce.

Reply:
Fun fact: The coalesce operator ?: is also known as the Elvis operator.